Reemit.Decompiler/ClrMethod.cs

```diff
@@ -0,0 +1,48 @@
+using Reemit.Decompiler.Clr.Metadata.Tables;
+using Reemit.Decompiler.Clr.Methods;
+
+namespace Reemit.Decompiler;
+
+public class ClrMethod(
+    string name,
+    int maxStack,
+    byte[] methodBody)
+{
+    public string Name { get; } = name;
+    public int MaxStack { get; } = maxStack;
+    public byte[] MethodBody { get; } = methodBody;
+
+    public static ClrMethod FromMethodDefRow(MethodDefRow methodDefRow, ModuleReaderContext context)
+    {
+        var name = context.StringsHeapStream.Read(methodDefRow.Name);
+        
+        // TODO: Investigate what exactly it means when a MethodDef row has its RVA set to 0
```

Comment:
Looks like there are a few cases where this can happen. Not sure if you're handling it this time around, but:

![image](https://github.com/v0idzz/Reemit/assets/7999934/8d91b2ea-0d3a-413a-93e5-dc5f8bf57779)